engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/fixed/FixedKafkaAvroSchemaProvider.scala

```diff
@@ -6,29 +6,31 @@ import org.apache.flink.api.common.typeinfo.TypeInformation
 import org.apache.flink.streaming.connectors.kafka.{KafkaDeserializationSchema, KafkaSerializationSchema}
 import pl.touk.nussknacker.engine.api.typed.typing
 import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryClient
-import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentSchemaRegistryClientFactory.ConfluentSchemaRegistryClient
+import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.ConfluentSchemaRegistryClientFactory.TypedConfluentSchemaRegistryClient
 import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.formatter.ConfluentAvroToJsonFormatter
 import pl.touk.nussknacker.engine.avro.schemaregistry.confluent.{ConfluentAvroDeserializationSchemaFactory, ConfluentAvroSerializationSchemaFactory, ConfluentSchemaRegistryClientFactory}
 import pl.touk.nussknacker.engine.avro.typed.AvroSchemaTypeDefinitionExtractor
 import pl.touk.nussknacker.engine.avro.{AvroUtils, KafkaAvroSchemaProvider}
 import pl.touk.nussknacker.engine.kafka.{KafkaConfig, RecordFormatter}
 
 /**
-  * This class has fake implementation of serializer and deserializer based on Confluent implementations - it's hack
-  * to serialization / deserialization schema without SchemaRegistry.
+  * This Provider based on Confluet implementation of serializer and deserializer. We use MockSchemaRegistryClient to
+  * mock fixed schema.
   *
-  * @TODO: In future we should create own serializator and deserializator for FixedSchema
+  * @TODO: In future we should create own serializator and deserializator for fixed schema
```

Comment:
What about adding notice "based on avro raw data without schema id. It will cause changing strategy of test data generation - it should use embedded schema". Without this notice it looks like this walk-around is for little problem that can be fixed instantly.

Reply:
Changed ;)